h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonWriteDriver.java

```diff
@@ -376,10 +376,12 @@ private void write(GeometryCollection coll, JsonGenerator gen) throws IOExceptio
      * @param gen
      * @throws IOException
      */
-    private void write(Polygon geom, JsonGenerator gen) throws IOException {
+    private void write(Polygon geom, JsonGenerator gen) throws IOException, SQLException {
         gen.writeStringField("type", "Polygon");
         gen.writeFieldName("coordinates");
         gen.writeStartArray();
+        ST_Transform transform = new ST_Transform;
+        geom = (Polygon) transform.ST_Transform(connection, geom, 4326);
```

Comment:
I'm not in favour of a geometry transformation. The geometry SRID must be managed by the user. It's up to the user to send the good CRS.